### ISSUE TITLE
Particle information interface

### ISSUE THREAD

**Issue by ibab** [ibab/missing_hep#3]
I could often make use of a way to retrieve particle information like masses, branching fractions etc. from scripts or from the IPython shell.
It would be ideal to have access to all the PDG data, including errors.

**Comment by alexpearce** [comment]
[PyPDT](https://pypi.python.org/pypi/PyPDT/0.2.0) is a good start, though doesn't include uncertainties (the numbers or [the package](https://pypi.python.org/pypi/uncertainties)).

The PDG database is notoriously hard to parse/scrape.
